ddtrace/logger.py

```diff
@@ -0,0 +1,38 @@
+import logging
+from logging.handlers import RotatingFileHandler
+import os
+
+
+DEFAULT_FILE_SIZE_BYTES = 15 << 20  # 15 MB
+
+
+def configure_ddtrace_logger():
+    # type: () -> None
+    """ """
+    debug_enabled = os.environ.get("DD_TRACE_DEBUG", "false").lower() in ("true", "1")
+    log_path = os.environ.get("DD_TRACE_LOG_FILE", None)
```

Comment:
```suggestion
    log_path = os.environ.get("DD_TRACE_LOG_FILE")
```

`None` is the default, not needed.